src/com/sun/jna/win32/W32APIOptions.java

```diff
@@ -6,18 +6,24 @@
 
 public interface W32APIOptions extends StdCallLibrary {
     /** Standard options to use the unicode version of a w32 API. */
-    Map UNICODE_OPTIONS = Collections.unmodifiableMap(new HashMap() {
+    Map<Object, Object> UNICODE_OPTIONS = Collections.unmodifiableMap(new HashMap<Object, Object>() {
```

Comment:
You are right - don't know why I thought otherwise. Fixed and pushed updated files.